packages/redux/src/reducers/api.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * JSAPI instance
+ */
+import { SET_API } from '../actionTypes';
+import { replaceReducer } from './common';
+
+export default replaceReducer(SET_API, null);
```

Comment:
If we set this to `undefined` instead of `null`, do we still need to make the changes to the formatters?

Reply:
No, the formatters changes aren't related to the Settings menu bug, changing `null` to `undefined` wouldn't make a difference. On a second thought, those changes weren't necessary, I've reverted the [Fix optional param initialization](https://github.com/deephaven/web-client-ui/pull/1292/commits/2ff1ed0a52f19ef8a20fef05f9b37c1a907bf387) commit.